src/components/ErrorMessage/ErrorMessage.tsx

```diff
@@ -0,0 +1,70 @@
+import React, { useEffect, useState } from 'react';
+import { AppError } from '../../types/AppError';
+
+interface Props {
+  removeErrorMessage: () => void;
+  reloadData: () => Promise<void>;
+  errorType: AppError;
+}
+
+export const ErrorMessage: React.FC<Props> = ({
+  removeErrorMessage,
+  reloadData,
+  errorType,
+}) => {
+  let message = '';
+
+  switch (errorType) {
```

Comment:
Why can't we write messages to object and read from it using error type? this will be simpler than switch/case